`-unsafe` compiler option required

To use HoloToolkit it is necessary to set the `-unsafe` compiler option. I'm new to C# so I'm not sure what this means, but I did find the following text on the Unity forums:

"Again, using unsafe doesn't give you much advantages, it just opens up for a huge variety of errors, mistakes and unexpected behaviour. Unless you know exactly what you do and you use pointers sparsely you shouldn't go for an unsafe context. " (http://answers.unity3d.com/questions/804103/how-to-enable-unsafe-and-use-pointers.html)

Is it wise to require unsafe? If this needs to remain then this should be documented in the readme (see link above for documentation)

Seems the above solution doesn't make the errors go away at build time. An example of the error I'm getting is "Assets\HoloToolkit\Assets\HoloToolkit\Sharing\Scripts\SDK\TagImage.cs(51,22): error CS0227: Unsafe code may only appear if compiling with /unsafe"

I've checked the "Allow unsafe code" box in Visual Studio, but that doesn't help.

--- a/GettingStarted.md
+++ b/GettingStarted.md
@@ -3,7 +3,12 @@ in your project.
 
 # Preparing the HoloToolkit-Unity Code
 
-`git clone https://github.com/Microsoft/HoloToolkit-Unity.git`
+We'll build the HoloToolkit from source since it's a reasonable assumption 
+that you want the latest and greatest if you are reading this document. To
+do this you need to clone the GitHUb repository at 
+https://github.com/Microsoft/HoloToolkit-Unity.git. If you've never cloned a
+repo before then you should consider using the GitHub desktop client, see 
+https://desktop.github.com/
 
 Open in Unity
 
@@ -17,10 +22,6 @@ Open or create your project in Unity
 you exported above]
 
 You should now have a `HoloToolkit` menu item.
-
-Since HoloToolkit uses "unsafe" code you need to tell Unity to compile 
-with the unsafe flag. To do this add a file `Assets/smcs.rsp` with the 
-contents `-unsafe`.
 
 # Preparing a Scene for Holographic Content
 

--- a/GettingStarted.md
+++ b/GettingStarted.md
@@ -1,3 +1,6 @@
+This document describes how to prepare the HoloToolkit sourcecode for use
+in your project.
+
 # Preparing the HoloToolkit-Unity Code
 
 `git clone https://github.com/Microsoft/HoloToolkit-Unity.git`

--- a/GettingStarted.md
+++ b/GettingStarted.md
@@ -1,0 +1,38 @@
+# Preparing the HoloToolkit-Unity Code
+
+`git clone https://github.com/Microsoft/HoloToolkit-Unity.git`
+
+Open in Unity
+
+`Assets -> Export Package…`
+
+# Using HoloToolkit-Unity in Your Project
+
+Open or create your project in Unity
+
+`Assets -> Import Package -> Custom Package…` [Navigate to the package 
+you exported above]
+
+You should now have a `HoloToolkit` menu item.
+
+# Preparing a Scene for Holographic Content
+
+Add the `Main Camera.prefab` and remove the default camera (this 
+creates a camera customized for holographic development)
+
+You will probably want to add `ManualCameraControl.cs` to the 
+`Main Camera`. This allows the user to manually control
+ the camera when in the Unity player.
+
+`HoloToolkit -> Configure -> Apply HoloLens Scene Settings`
+
+`HoloToolkit -> Configure -> Apply HoloLens Project Settings`
+
+# Building Your Project for HoloLens
+
+`HoloToolkit -> Builds -> Build for HoloLens`
+
+Now open in Visual Studio and deploy to the emulator or device/
+
+
+

--- a/GettingStarted.md
+++ b/GettingStarted.md
@@ -18,6 +18,10 @@ you exported above]
 
 You should now have a `HoloToolkit` menu item.
 
+Since HoloToolkit uses "unsafe" code you need to tell Unity to compile 
+with the unsafe flag. To do this add a file `Assets/smcs.rsp` with the 
+contents `-unsafe`.
+
 # Preparing a Scene for Holographic Content
 
 Add the `Main Camera.prefab` and remove the default camera (this 

--- a/README.md
+++ b/README.md
@@ -12,6 +12,8 @@ HoloToolkit contains the following feature areas:
 
 To learn more about individual HoloLens feature areas, please read the [Wiki](https://github.com/Microsoft/HoloToolkit-Unity/wiki) section.
 
+To learn how to add the HoloToolkit to your project see the [Getting Started](GettingStarted.ms) guide.
+
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). 
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 

--- a/README.md
+++ b/README.md
@@ -12,7 +12,7 @@ HoloToolkit contains the following feature areas:
 
 To learn more about individual HoloLens feature areas, please read the [Wiki](https://github.com/Microsoft/HoloToolkit-Unity/wiki) section.
 
-To learn how to add the HoloToolkit to your project see the [Getting Started](GettingStarted.ms) guide.
+To learn how to add the HoloToolkit to your project see the [Getting Started](GettingStarted.md) guide.
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). 
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.

